app/src/main/java/org/oppia/app/FakeDataProvider.kt

```diff
@@ -5,7 +5,7 @@ import com.squareup.moshi.Moshi
 import org.oppia.data.backends.gae.model.GaeExplorationContainer
 
 // Remove this file at later stage once domain and model module are working.
-class FakeDataProvider() {
+class FakeDataProvider {
   companion object {
```

Comment:
You could probably move the constant up to be file-level, or use a top-level object, e.g.:

```kotlin
object FakeDataProvider {
  const val ...
}
```

Reply:
This file has been removed because though we had created this file for fake data use but write now we don't need this as we are testing everything directly from another fragment/activity.